cli/Cargo.toml

```
@@ -66,17 +66,17 @@ deno_ast = { workspace = true, features = ["bundler", "cjs", "codegen", "proposa
 deno_cache_dir = { workspace = true }
 deno_config = "=0.15.0"
 deno_core = { workspace = true, features = ["include_js_files_for_snapshotting"] }
-deno_doc = { version = "=0.123.1", features = ["html"] }
-deno_emit = "=0.39.0"
-deno_graph = { version = "=0.71.5", features = ["tokio_executor"] }
-deno_lint = { version = "=0.58.2", features = ["docs"] }
+deno_doc = { version = "0.124.0", features = ["html"] }
+deno_emit = "0.39.1"
+deno_graph = { version = "0.72.0", features = ["tokio_executor"] }
+deno_lint = { version = "0.58.3", features = ["docs"] }
```

```suggestion
deno_doc = { version = "=0.124.0", features = ["html"] }
deno_emit = "=0.39.1"
deno_graph = { version = "=0.72.0", features = ["tokio_executor"] }
deno_lint = { version = "=0.58.3", features = ["docs"] }
```

There have been accidental breaking changes in these crates which breaks people doing `cargo install`. It's better to pin these because they're only used in the cli.